lucidsonicdreams/main.py

```diff
@@ -82,14 +96,49 @@ def __init__(self,
     self.num_possible_classes = num_possible_classes 
     self.style_exists = False
     
+    # some stylegan models cannot be converted to pytorch (wikiart)
+    self.use_tf = style in ("wikiart",)
```

Comment:
I tried using 'abstract photos' and it was not working as well.

Error:

```
Loading networks from abstract photos.pkl...
Traceback (most recent call last):
  File "script.py", line 6, in <module>
    L.hallucinate(file_name="song.mp4")
  File "/Users/julienbeisel/Documents/git repos/dev/lucid-sonic-dreams/lucidsonicdreams/main.py", line 702, in hallucinate
    self.stylegan_init()
  File "/Users/julienbeisel/Documents/git repos/dev/lucid-sonic-dreams/lucidsonicdreams/main.py", line 182, in stylegan_init
    self.Gs = self.legacy.load_network_pkl(f)['G_ema'].to(device) # type: ignore
  File "stylegan2/legacy.py", line 26, in load_network_pkl
    G = convert_tf_generator(tf_G)
  File "stylegan2/legacy.py", line 111, in convert_tf_generator
    raise ValueError('TensorFlow pickle version too low')
ValueError: TensorFlow pickle version too low
```

I am going to try to convert it differently since other models seem to work.